src/administration/snapshot-and-restore.md

```diff
@@ -42,25 +41,41 @@ Finding snapshots for the environment master
 ```
 
 You can restore a specific snapshot with the CLI as follows:
+
 ```bash
 $ platform snapshot:restore 92c9a4b2aa75422efb3d
 ```
 
 > **note**
 > You need "admin" role to restore your environment from a snapshot.
 
-### Automated snapshots
+## Automated snapshots
 
-No snapshot is triggered automatically on Platform.sh Standard. You can
-trigger your snapshot via the Web Interface or via the CLI.
+No snapshot is triggered automatically on Platform.sh Standard. You can trigger your snapshot via the Web Interface or via the CLI.
 
-If you want to automate your snapshots, you can use Jenkins and trigger
-with the CLI command as follows:
+Snapshots may be triggered from an automated system, such as cron, Jenkins, or another CI service by calling the CLI.  To enable backups automatically using just Platform.sh, see the section on [API tokens](/gettingstarted/cli/api-tokens.md).  For installing the CLI in the app container.
 
-```bash
-$ platform snapshot:create --yes --no-wait
+Once the CLI is installed an API token configured you can add a cron task to run once a day and trigger a snapshot.  The CLI will read the existing environment variables in the container and default to the project and environment it is running on.  However, in most cases such backups are only useful on the `master` production environment.  That can be achieved like so:
```

Comment:
Once the CLI "has" installed?

Reply:
Nope, missing an "and" later in the sentence. :smile: